openhtf/output/web_gui/src/update_plugins.py

```diff
@@ -99,20 +99,18 @@
 import 'file?name=/templates/station.html!./station.html';
 """
 
-mid1_ts='''
+post_ts="""
 
 /** Station view component. **/
-@Component({
+@Component({{
```

Comment:
Oh, if you used `%(plugin_configs)s` you wouldn't need to double up all these curly braces, sorry!